src/System.Text.Json.Dynamic/System/Text/Json/JsonDynamicObject.cs

```diff
@@ -167,8 +177,9 @@ public override bool TrySetMember(SetMemberBinder binder, object value)
         public bool TryFormat(Span<byte> buffer, out int written, TextFormat format, EncodingData formattingData)
         {
             written = 0;
+            int consumed;
             int justWritten;
-            if(!formattingData.TextEncoder.TryEncode('{', buffer, out justWritten)) {
+            if(!formattingData.TextEncoder.TryEncode(OpenCurlySpan, buffer, out consumed, out justWritten)) {
```

Comment:
From what live code to this change, it is equal or faster with change. The reason is that TryEncode with a single char just wrapped the char in a span and called the span version anyways.